extras.py

```diff
@@ -23,7 +24,14 @@ def __init__(self, tower, enemy, board, cellDim):
         self.center = self.calculateCenter(self.location)
 
         pygame.mixer.init()
-        pygame.mixer.music.load("pew.wav")  # Loading File Into Mixer
+        try:
+            # PyInstaller creates a temp folder and stores
+            # path in _MEIPASS
+            base_path = sys._MEIPASS  # noqa
```

Comment:
It's not OS related, it's pyinstaller related.
So if you're trying to create executable using pyinstaller, the assignment in the try will succeed, otherwise it will fail and just get the normal path, and that's what you want to happen no matter what OS you're using.

Reply:
It's not OS related, it's pyinstaller related.
So if you're trying to create executable using pyinstaller, the assignment in the try will succeed, otherwise it will fail and just get the normal path, and that's what you want to happen no matter what OS you're using.